src/main/java/com/plaid/client/model/payment/Recipient.java

```diff
@@ -0,0 +1,27 @@
+package com.plaid.client.model.payment;
+
+import com.plaid.client.model.payment.Address;
+
+public final class Recipient {
```

Comment:
The API schema also returns `recipientId` here

Reply:
good catch ⚡️opening a new PR to address this